common/src/main/proto/TransportMessages.proto

```diff
@@ -444,7 +444,9 @@ message PbHeartbeatFromApplication {
   repeated PbWorkerInfo needCheckedWorkerList = 5;
   bool shouldResponse = 6;
   int64 shuffleCount = 7;
-  map<string, int64> shuffleFallbackCounts = 8;
+  int64 applicationCount = 8;
+  map<string, int64> shuffleFallbackCounts = 9;
```

Comment:
Sorry, I am using the main branch for deployment, it is not necessary to change the order here.
```
map<string, int64> shuffleFallbackCounts = 8;
int64 applicationCount = 9;
```

Reply:
And the order looks fine.
```
shuffleCount
shuffleFallbackCounts
applicationCount
applicationFallbackCounts
```